tests/kernel/ibtests.pm

```diff
@@ -87,7 +87,7 @@ sub ibtest_master {
     # wait until the two machines under test are ready setting up their local things
     assert_script_run('cd hpc-testing');
     barrier_wait('IBTEST_BEGIN');
-    assert_script_run("./ib-test.sh $args $master $slave", $timeout);
+    script_run("./ib-test.sh $args $master $slave", $timeout);
```

Comment:
Why would you want to change this? In case this call fails, the test should die and execute the post fail hook, shouldn't it?

Reply:
i dont know exactly whats going on but the ibtest destructs the system or something and VPN connection dies and the `assert_script_run` never gets the return code. in case that the call fails, the test will die in the next command or it will be logged!

Reply:
I think we can do this. But I think we can remove some then obsolete duplicate code from the post_fail_hook, I think.

Reply:
Done. i run new VR

Reply:
*this request is ready for review*